Source/UETorch/Private/UETorch.cpp

```diff
@@ -706,3 +706,43 @@ extern "C" bool CaptureDepthField(UObject* _this, const IntSize* size, void* dat
 	}
 	return true;
 }
+
+extern "C" bool SetActorPosition(AActor* object, double x, double y, double z) {
+	return object->SetActorLocation(FVector(x,y,z), false);
+}
+
+extern "C" bool SetActorRotation(AActor* object, double pitch, double roll, double yaw) {
+	return object->SetActorRotation(FRotator(pitch,roll,yaw));
+}
+
```

Comment:
Add `SetActorLocationAndRotation` while you're at it, it works better than setting them individually (I'll explain some other time).